recipes/easyexif/all/conanfile.py

```diff
@@ -15,44 +19,36 @@ class EasyExifConan(ConanFile):
         "fPIC": [True, False]
     }
     default_options = {"shared": False, "fPIC": True}
-    generators = "cmake"
-    
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
 
     def config_options(self):
         if self.settings.os == "Windows":
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
         if self.settings.compiler.cppstd:
```

Comment:
we should always raise from validate :)

```suggestion

    def validate(self):
        if self.settings.get_safe("compiler.cppstd"):
```